packages/angular/src/app/pages/analytics-sales-report/analytics-sales-report.component.html

```diff
@@ -77,11 +77,11 @@
       </dx-chart>
     </card-analytics>
 
-    <card-analytics titleText="Sales Performance" className="sales">
+    <card-analytics titleText="Sales Performance">
       <div filter class="sales-filter" >
-      <dx-drop-down-button stylingMode="text" [useSelectMode]="true"
-        [items]="groupByPeriods" [selectedItemKey]="groupByPeriods[1]" (onSelectionChanged)="selectionChange($event)">
-      </dx-drop-down-button>
+        <dx-drop-down-button stylingMode="text" [useSelectMode]="true"
+                             [items]="groupByPeriods" [selectedItemKey]="groupByPeriods[1]" (onSelectionChanged)="selectionChange($event)">
```

Comment:
whats this tab for?

Reply:
it's github's formatting, it's not in the source